src/screens/Notification.js

```diff
@@ -0,0 +1,314 @@
+          <Description>Any valid Text prop for the title text.</Description>
+          <PropertyValue type="object">
+            <Description>
+              'color' is used for the text color and 'weight' for the weight of
+              the title.
+            </Description>
+            <Example defaultValue>
+              {`
+                    {
+                        weight: 'bold',
+                        color: {
+                            light: 'black',
+                            dark: 'white',
+                        },
+                    },
+                `}
+            </Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="notification.message">
+          <Description>Any valid Text prop for the message text.</Description>
+          <PropertyValue type="object">
+            <Description>'color' is used for the message color.</Description>
+            <Example defaultValue>
+              {`
+                    {
+                        color: {
+                            light: 'black',
+                            dark: 'white',
+                        },
+                    },
+                `}
+            </Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="notification.button.icon">
+          <Description>The default close button icon.</Description>
+          <PropertyValue type="node">
+            <Example defaultValue>{`<FormClose />`}</Example>
+            <Example>{`<Icon />`}</Example>
+          </PropertyValue>
+          <PropertyValue type="SVG">
+            <Example>{`<svg>...</svg>`}</Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="notification.button.color">
+          <Description>The color of the close button icon.</Description>
+          <GenericColor />
+          <PropertyValue>
+            <Example defaultValue>
+              {`
+                    color: {
+                        light: 'black',
+                        dark: 'white',
+                    },`}
+            </Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="notification.unknown.icon">
+          <Description>
+            The default icon for an unknown status Notification.
+          </Description>
+          <PropertyValue type="node">
+            <Example defaultValue>{`<StatusUnknownSmall />`}</Example>
+            <Example>{`<Icon />`}</Example>
+          </PropertyValue>
+          <PropertyValue type="SVG">
+            <Example>{`<svg>...</svg>`}</Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="notification.unknown.color">
+          <Description>The color of the close button icon.</Description>
+          <GenericColor />
+          <PropertyValue>
+            <Example defaultValue>
+              {`
+                    "status-unknown"
+                `}
+            </Example>
```

Comment:
Same alignment comment as above. Show "status-unknown" as the string example instead of "brand". Also PropertyValue should have a type prop.